packages/deploy/src/api/platform.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseApiClient } from 'defender-base-client';
+
+export class PlatformApiClient extends BaseApiClient {
+  protected getPoolId(): string {
+    return process.env.DEFENDER_ADMIN_POOL_ID || 'us-west-2_94f3puJWv';
+  }
+
+  protected getPoolClientId(): string {
+    return process.env.DEFENDER_ADMIN_POOL_CLIENT_ID || '40e58hbc7pktmnp9i26hh5nsav';
+  }
+
+  protected getApiUrl(): string {
+    return process.env.DEFENDER_ADMIN_API_URL || 'https://defender-api.openzeppelin.com/admin/';
```

Comment:
is this under `admin`? Seems like it would be `deployments` or something. Maybe admin was intended?

Reply:
Yep you are right it should be deployments. I have we anything put in place for mapping on prod or is it going to be a manual process?

Reply:
you're going to need to add a `deployments` mapping in prod similar to the others if the URL is `${baseUrl}/deployments/...`